charts/gitlab-ci-runner-exporter/templates/deployment.yaml

```
@@ -65,6 +65,8 @@ spec:
               value: :{{ .Values.containerPorts.metrics }}
             - name: LOG_LEVEL
               value: {{ .Values.gcpe.logLevel }}
+
+            {{ toYaml .Values.envVariables |  nindent 6 }}
```

```suggestion
            {{- toYaml .Values.envVariables | nindent 12 }}
```

You can test the render of the chart using following command :slightly_smiling_face: 
```sh
helm template ./charts/gitlab-ci-runner-exporter --debug
```